wombat-admin/src/components/navbar.tsx

```diff
@@ -17,32 +21,42 @@ export default function Navbar() {
                 <div className="ml-10 flex items-baseline space-x-4">
                   {/* <!-- Current: "bg-teal-700 text-white", Default: "text-white hover:bg-teal-500 hover:bg-opacity-75" --> */}
                   <a
-                    href="#"
-                    className="bg-teal-700 text-white rounded-md px-3 py-2 text-sm font-medium"
+                    href="/"
```

Comment:
I think it'd be better to use NextJS' Link component here instead of <a> tags. https://nextjs.org/docs/app/api-reference/components/link